src/main/java/ch/njol/skript/util/FileUtils.java

```diff
@@ -60,7 +63,32 @@ public static String getBackupSuffix() {
 			return "" + backupFormat.format(System.currentTimeMillis());
 		}
 	}
-	
+
+	/**
+	 * Deletes files in backup directory to meet desired target, starting from oldest to newest
+	 *
+	 * @param csvFile Variable file in order to get 'backups' directory
+	 * @param toKeep Integer of how many files are to be left remaining
+	 * @throws IOException If 'backups' directory is not found
+	 * @throws IllegalArgumentException If 'toKeep' parameter is less than 0
+	 */
+	public static void backupPurge(File csvFile, int toKeep) throws IOException, IllegalArgumentException {
```

Comment:
Since this is only FlatFile CSV related, can this go within the FlatFileStorage class?